cloud-controller-manager/cmd/digitalocean-cloud-controller-manager-admission-server/Dockerfile

```diff
@@ -0,0 +1,21 @@
+# Copyright 2023 DigitalOcean
```

Comment:
We need to update the year at this point (other files as well).

Reply:
Will be addressed in a separaete PR.